lib/clusters/iasZone.js

```diff
@@ -1,11 +1,61 @@
 'use strict';
 
 const Cluster = require('../Cluster');
+const { ZCLDataTypes } = require('../zclTypes');
 
 const ATTRIBUTES = {
+  zoneState: {
+    id: 0,
+    type: ZCLDataTypes.enum8({
+      notEnrolled: 0,
+      enrolled: 1,
+    }),
+  },
+  zoneType: {
+    id: 1,
+    type: ZCLDataTypes.enum8({
+      standardCIE: 0,
+      motionSensor: 13,
+      contactSwitch: 21,
+      fireSensor: 40,
+      waterSensor: 42,
+      cabonMonoxideSensor: 43,
+      personalEmergencyDevice: 44,
+      vibrationMovementSensor: 45,
+      remoteControl: 271,
+      keyfob: 277,
+      keypad: 541,
+      standardWarningDevice: 549,
+      glassBreakSensor: 550,
+      securityRepeaters: 553,
+      invalidZoneType: 65535,
+    }),
+  },
+  zoneStatus: {
+    id: 2,
+    type: ZCLDataTypes.map16('alarm1', 'alarm2', 'tamper', 'battery', 'supervisionReports', 'restoreReports', 'trouble', 'acMains', 'test', 'batteryDefect'),
+  },
+  iasCIEAddress: {
+    id: 16,
+    type: ZCLDataTypes.EUI64,
+  },
+  zoneID: {
+    id: 17,
+    type: ZCLDataTypes.uint8,
+  },
 };
 
-const COMMANDS = {};
+const COMMANDS = {
+  zoneStatusChangeNotification: {
+    id: 0,
+    args: {
+      zoneStatus: ZCLDataTypes.map16('alarm1', 'alarm2', 'tamper', 'battery', 'supervisionReports', 'restoreReports', 'trouble', 'acMains', 'test', 'batteryDefect'),
```

Comment:
This should be moved up so it can be shared with the zoneStatus attribute (since they must be identical).

Reply:
Not sure what you mean here